pkg/kubelet/cm/cpumanager/state/state_checkpoint_test.go

```diff
@@ -227,8 +228,9 @@ func TestCheckpointStateRestore(t *testing.T) {
 			restoredState, err := NewCheckpointState(testingDir, testingCheckpoint, tc.policyName, tc.initialContainers)
 			if err != nil {
 				if strings.TrimSpace(tc.expectedError) != "" {
-					tc.expectedError = "could not restore state from checkpoint: " + tc.expectedError
-					if strings.HasPrefix(err.Error(), tc.expectedError) {
+					tc.expectedError = "could not restore state from checkpoint: " + "(.*)" + tc.expectedError
```

Comment:
expectedError is not intended to be a regex (the messages have special regex characters like `.`)... would it be sufficient to check `strings.Contains(err.Error(), "could not restore state from checkpoint") && strings.Contains(err.Error(), tc.expectedError)`?

Reply:
Done